lib/system.nim

```diff
@@ -4198,6 +4198,12 @@ type
   NimNode* {.magic: "PNimrodNode".} = ref NimNodeObj
     ## Represents a Nim AST node. Macros operate on this type.
 
+proc lenVarargsImpl(x: NimNode): NimNode {.magic: "LengthOpenArray", noSideEffect.}
+
+macro lenVarargs*(x: varargs[untyped]): int {.magic: "LengthOpenArray", noSideEffect.} =
```

Comment:
Why does this have the ``.magic`` too?

Reply:
good catch, fixed (compiler should maybe disallow .magic in non-proc contexts)